docs-content/index.html.md.erb

```diff
@@ -70,10 +70,22 @@ The following table provides version and version-support information about Cyber
 CyberArk Conjur Service Broker for VMware Tanzu has the following requirements:
 
 + **You must have an existing Conjur instance installed.  The Conjur instance may be external to the VMware Tanzu environment. Supported versions are:**
+  
+  * CyberArk Dynamic Access Provider (DAP), v10.9 and later
   * Conjur Enterprise (v4.9.12.0 and later)
       * Use v4.9.17.0 or later to search for CF hosts in the Conjur UI
-  * Conjur Open Source (v1.0 and later)
-      * Hosted Conjur is a hosted version of Open Source Conjur. You can quickly get a hosted Conjur instance running for evaluation purposes. Visit the [hosted Conjur page](https://www.conjur.org/get-started/try-conjur.html) to sign up.
+  * Conjur Open Source, v1.0 and later
+      * We **strongly** recommend choosing the version of this project to use from the latest [Conjur OSS
+suite release](https://docs.conjur.org/Latest/en/Content/Overview/Conjur-OSS-Suite-Overview.html).
+Conjur maintainers perform additional testing on the suite release versions to ensure
+compatibility. When possible, upgrade your Conjur version to match the
+[latest suite release](https://docs.conjur.org/Latest/en/Content/ReleaseNotes/ConjurOSS-suite-RN.htm);
+when using integrations, choose the latest suite release that matches your Conjur version. For any
+questions, please contact us on [Discourse](https://discuss.cyberarkcommons.org/c/conjur/5).
```

Comment:
is this going to resolve correctly when published if it doesn't have the indentation under the bullet?

Reply:
Indeed, I posted some PDFs of the pages I changed. It may not look pretty un-rendered, but it'll be rendered correctly.